apps/greenhouse-management/src/lib/store.js

```diff
@@ -23,6 +23,8 @@ export default (options) => {
 
   // set the endpoint and embedded props for the management plugin coming from the package.json
   Object.keys(configs).forEach((key) => {
+    // pull latest version in dev and qa
+    configs[key].version = options.environment =='qa' || options.environment == 'dev' ? 'latest' : configs[key].version
```

Comment:
In the description of the appProp, the environment is exemplified as `development`. The same convention is also followed in the Greenhouse app. Wouldn't it make sense to use `development` in the check or add it to the check?

Reply:
:100: fixed